File: Volume 10 (1000 - 1099)/1038/1038.c
#include <stdio.h>

int main() {
    int a, b;
    double l1[6] = {0, 4, 4.5, 5, 2, 1.5};
    scanf("%d %d", &a, &b);
    printf("Total: R$ %.2f\n", b * l1[a]);
    return 0;
}

File: Volume 10 (1000 - 1099)/1038/1038.py
#!/usr/bin/env python3

l1 = [0, 4, 4.5, 5, 2, 1.5]
l2 = [int(i)for i in input().split(' ')]
print("Total: R$ {:.2f}".format(l2[1] * l1[l2[0]]))